src/main/groovy/io/micronaut/build/docs/CreateReleasesDropdownTask.java

```diff
@@ -54,17 +62,16 @@ public abstract class CreateReleasesDropdownTask extends DefaultTask {
     @OutputFile
     public abstract RegularFileProperty getOutputIndex();
 
-    @Input
-    public abstract Property<String> getVersionsJson();
```

Comment:
:x: If you remove this, then the versions JSON is no longer an input to the task. It will be considered up-to-date (and cached!) even if the list of versions change. That's why it's using a provider.

Reply:
how about https://github.com/micronaut-projects/micronaut-build/pull/454/commits/868b13381ed116db7df0a892740815006445b9de